developers/prerequisites/osgids.md

```diff
@@ -290,7 +295,22 @@ For more information - [OSGi 4.2 Compendium Specifiaction, Chapter 112.5 Compone
 
 If you want to practice, what you have learned so far, you can try out our [OSGi Coding tasks](osgitasks.html). In chapter IV. there are tasks, that are using OSGi DS. It is recommended to start from the beginning (chapter I.), as the tasks are related between each other.
 
-## VIII. Further Reading
+## VIII. Troubleshooting
+
+If you are new to Declarative Services and you don't know how to troubleshoot, maybe we can help you. We will give you a few tips what might have gone wrong. Some of the steps below depend on your development IDE and the OSGi Conainer that you use, to get most of them, please use the Eclipse IDE and Equinox as the OSGi Runtime. You might review again the [Equinox commands](equinox.html#iv-commands) before you continue:
```

Comment:
> ...Conainer that you use, to get most of them, please use the Eclipse IDE and Equinox as the OSGi Runtime....

This seems a bit unclear to me and the sentence is too long and complicated. I think that in order to make the clear we can do it as follows:
...Conainer that you use. Mind that most of the tips below are specific to Eclipse IDE and Equinox as the OSGi Runtime.